sphinxcontrib/katex.py

```diff
@@ -356,23 +356,26 @@ class KaTeXError(Exception):
 class KaTeXServer:
     """Manages and communicates with an instance of the render server."""
 
-    # Message length is 32-bit little-endian integer
-    LENGTH_STRUCT = struct.Struct("<i")
+    length_struct = struct.Struct("<i")
+    """Message length (32-bit little-endian integer)."""
```

Comment:
You did not have to change LENGTH_STRUCT and STOP_TIMEOUT to lowercase, since they are constants (not changed anywhere in the code).

Reply:
Good point, I reverted those changes.